app/models/approved_upload_snapshot.rb

```diff
@@ -4,7 +4,8 @@ def finalize_upload
     new_files.each do |file|
       work.track_change(:added, file["filename"])
     end
-    WorkActivity.add_work_activity(work.id, "#{new_files.count} #{'file'.pluralize(new_files.count)} were moved to the post curation bucket", new_files.first["user_id"],
+    target_location = work.embargoed? ? "embargo" : "post-curation"
```

Comment:
@hectorcorrea can this code utilize `work.files_mode_human`?

Reply:
@carolyncole sadly no, the work state is not yet approved here so I had to just check the embargo here (and not the state)